airbyte-integrations/connectors/third-party/propel/airbyte-propel-destination/metadata.yaml

```diff
@@ -0,0 +1,24 @@
+data:
+  registries:
+    cloud:
+      enabled: false
+    oss:
+      enabled: true
+  connectorSubtype: api
+  connectorType: destination
+  definitionId: 938e4f36-ab42-4486-9a21-d43df92ed9e4
+  dockerImageTag: 0.0.1
+  dockerRepository: propeldata/airbyte-propel-destination
+  githubIssueLabel: propeldata/airbyte-propel-destination
+  icon: propel.svg
+  license: MIT
+  name: Propel
+  releaseStage: alpha
+  documentationUrl: https://docs.airbyte.com/integrations/destinations/propel
+  tags:
+    - language:unknown
+  ab_internal:
```

Comment:
I don't know if these make sense — and I'm not sure why we default them to 100.

Reply:
Same as above, the others had it as 100 😕 but let me know if those should change